x-pack/filebeat/module/cisco/shared/ingest/asa-ftd-pipeline.yml

```diff
@@ -253,7 +253,7 @@ processors:
   - dissect:
       if: "ctx._temp_.cisco.message_id == '106100'"
       field: "message"
-      pattern: "access-list %{_temp_.cisco.list_id} %{event.outcome} %{network.transport} %{_temp_.cisco.source_interface}/%{source.address}(%{source.port}) -> %{_temp_.cisco.destination_interface}/%{destination.address}(%{destination.port}) %{}"
+      pattern: "access-list %{_temp_.cisco.list_id} %{event.outcome} %{network.transport} %{_temp_.cisco.source_interface}/%{source.address}(%{source.port})%{}-> %{_temp_.cisco.destination_interface}/%{destination.address}(%{destination.port})%{}"
```

Comment:
Did we want to add the ECS mapping to the user if we can identify the field as such? As [indicated by the docs](https://www.cisco.com/c/en/us/td/docs/security/asa/syslog/b_syslog/syslogs1.html#con_4769049) the thing we were missing is one of the following:

>  idfw_user— The user identity username, including the domain name that is added to the existing syslog when the ASA can find the username for the IP address.
sg_info— The security group tag that is added to the syslog when the ASA can find a security group tag for the IP address. The security group name is displayed with the security group tag, if available. 

If we just want this as a bugfix to not drop the event that's cool too, as I'm not entirely sure how to distinguish `idfw_user` v. `sg_info`.

Reply:
Yeah, it's both issues, didn't want to complicate the parsing even more for a bug fix, and I don't see an easy way of telling user and info apart.